ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -120,6 +125,7 @@ public boolean accept(File dir, String name) {
     this.sessionId = sessionId;
     this.incarnationId = incarnationId;
     fixStateIfRequired();
+    bundleReadBuffer = ByteBuffer.allocate(config.storeCleanupOperationsBufferSize);
```

Comment:
I wonder if we should use `ByteBuffer.allocateDirect(...)` in this case since the buffer is long-lived and it could potentially enable some optimizations when used with `FileChannel`

https://stackoverflow.com/questions/5670862/bytebuffer-allocate-vs-bytebuffer-allocatedirect